README.md

```diff
@@ -1,20 +1,76 @@
 # ExBackend
 
-To start your Phoenix server:
+At this moment, the backend is composed of 2 parts:
+
+1. Back-end in Elixir which provide API,
+2. Front-End in Angular which consume API from back-end.
+
+## Requirements
+
+The following tool must be installed on your computer:
+
+* Elixir development environment (see installation [here](https://elixir-lang.org/install.html))
+
+## Back-End
+
+### Installation
+
+Backend installation:
 
   * Install dependencies with `mix deps.get`
   * Create and migrate your database with `mix ecto.create && mix ecto.migrate`
-  * Install Node.js dependencies with `cd assets && npm install`
-  * Start Phoenix endpoint with `mix phx.server`
+  * To run the application locally in dev mode, in the config folder, you need to create a file `dev.secret.exs` which will contains `use Mix.Config`
+
+### Start back-end server locally
+
+Start Phoenix endpoint with `mix phx.server`
+
+## Front-End
+
+### Installation 
+
+Install Node.js dependencies with : `cd assets && npm install`
+
+### Launch front server locally
 
 Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
 
-Ready to run in production? Please [check our deployment guides](http://www.phoenixframework.org/docs/deployment).
+## Makefile targets
+
+Commands below will be used for both stacks (backend & workers):
+
+| Command                     | Description                                                                                |
+|-----------------------------|--------------------------------------------------------------------------------------------|
+| `make docker-build`         | Build locally a docker image                                                               |
+| `make docker-clean`         | Remove locally the built docker image                                                      |
+| `make docker-push-registry` | Push the locally built docker image                                                        |
+| `make up`                   | Launch both, the back and front end locally [http://localhost:4000](http://localhost:4000) |
 
-## Learn more
+## Environment variables
+
+Some variables are defined to apply a custom setting. These variables are:
+
+| Variable name          | Default value                | Description                                     |
```

Comment:
can we put it into a sub markdown file ?

Reply:
I don't understand. Do you suggest to extract environment variables table and put them into a specific markdown file?

Reply:
Yes we can use an another file named (for example) variables.md.
And in this markdown we can link to it ...
it becomes very long file, and not very sexy for beginners...

Reply:
Ok, I've created a new folder named `docs` and I've stored `variables.md` in it.
This will be usefule later when documentation will grown.